### PR TITLE
Run recalculate_stagnant_cases only on icds envs and india

### PR DESCRIPTION
**Pull request by dannyroberts**
This error https://sentry.io/organizations/dimagi/issues/1107518919/?project=136860&query=is%3Aunresolved+StaticDataSourceConfigurationNotFoundError&statsPeriod=14d shows up quite a bit. Don't think it causes much harm but it was annoying me in sentry and it's an easy fix.

### CODE REVIEW

File: custom/icds_reports/tasks.py [hunk 1]
```diff
@@ -704,7 +696,8 @@ def email_dashboad_team(aggregation_date, aggregation_start_time, force_citus=Fa
     icds_data_validation.delay(aggregation_date)
 
 
-@periodic_task(
+@periodic_task_on_envs(
+    settings.ICDS_TESTING_ENVS,
```
Comment (snopoke) [inline custom/icds_reports/tasks.py]: This name is misleading. I thought it was only the 'testing' envs but it's actually all ICDS envs. Maybe we should make it 'ALL_ICDS_ENVS` and `PRIMARY_ICDS_ENVS`?

Also we can remove `icds-new`

**Review by snopoke — APPROVED** [review]
one comment but not blocking

File: custom/icds_reports/tasks.py (hunk 1 above)
Comment (emord) [reply custom/icds_reports/tasks.py]: I think we can just use `ICDS_ENVS`. I don't htink that the stagnant cases recalculation is important on india
```suggestion
    settings.ICDS_ENVS,
```

Comment (dannyroberts) [reply custom/icds_reports/tasks.py]: OK I'll take this suggestion and get rid of `ICDS_TESTING_ENVS`

Comment (dannyroberts) [reply custom/icds_reports/tasks.py]: Point taken on the naming though. If I do end up needing to include india on one of these in the future, I'll use something like what you suggested, Simon

**Pull request merged**